framework/console/controllers/BaseMigrateController.php

```diff
@@ -365,6 +404,34 @@ public function actionMark($version)
     }
 
     /**
+     * Checks if given migration version specification matches namespaced migration name.
+     * @param string $rawVersion raw version specification received from user input.
+     * @return string|false actual migration version, `false` - if not match.
+     * @since 2.0.10
+     */
+    private function extractNamespaceMigrationVersion($rawVersion)
+    {
+        if (preg_match('/^\\\\?([\w_]+\\\\)+m(\d{6}_?\d{6})(\D.*?)?$/is', $rawVersion, $matches)) {
```

Comment:
It just follows the original implementaton. It is enough to specify version timeline using numbers, instead of writing full class name.
Can not be changed otherwise it will cause BC break.